src/en/charms-upgrading.md

```diff
@@ -21,6 +21,25 @@ You can query for charm information, such as channels and revisions, with the
 `charm` utility, which is available via the Charm Tools software. See the
 [Charm Tools][charm-tools] page for guidance.
 
+## Crossgrading an application
+
+Crossgrading an application refers to upgrading an application by switching out
+the current charm and replacing it with a new local charm. This is accomplished
+by way of the `--switch` option. This differs from upgrading an existing charm
+locally (with the `--path` option) due to the charm being considered entirely
+new.
+
+This is considered a dangerous operation since Juju has only limited
+information with which to determine compatibility. The operation will succeed
+so long as the following conditions are met:
+
+- The new charm must declare all relations that the application is currently
```

Comment:
I know this is technically correct, but it I think it could be misunderstood. Perhaps " the new charm must support..." or "be compatible with"

Reply:
OK